new/property-defaults.md

```diff
@@ -0,0 +1,120 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Property Default Value Improvements
+
+This RFC addresses some longstanding issues with `default` values on properties (attributes), adds `reset_property` to Resource, and addresses how we will move core Chef resources over to use defaults and properties.
+
+## Motivation
+
+    As a Chef developer,
+    I want the core resources to use `property`,
+    So that people are cribbing off good examples.
+
+    As a Chef user,
+    I want the core resources not to actually change behavior,
+    So that I can rely on things I've built on top of them.
+
+## Specification
+
+### Non-Sticky Defaults
+
+Defaults are presently *sticky*, in that the first time a property is accessed, it is stored in the instance variable:
+
+```ruby
+class X < Chef::Resource
+  attribute :foo, default: lazy { Random.rand }
```

Comment:
I don't think I see the `attribute` anywhere in https://github.com/chef/chef/blob/12-stable/lib/chef/resource.rb - does this exist currently in this class? I thought it had to be subclassed from LWRPBase.